src/commands.cpp

```diff
@@ -4834,23 +4846,70 @@ bool CommandSetPH::procresult(Result r)
         return true;
     }
 
-    handle ph = client->json.gethandle();
 
-    if (ISUNDEF(ph))
+    if (mWritable) // aparently, depending on 'w', the response can be [{"ph":"XXXXXXXX","w":"YYYYYYYYYYYYYYYYYYYYYY"}] or simply [XXXXXXXX]
```

Comment:
That's something we should clarify. I'm quite sure that the API returns the `ph` and `w` when the link is created, and only the public handle when the link already exists.